versioned_docs/version-5.1/general-concepts/multilingual.md

```diff
@@ -1,3 +1,193 @@
 Multilingual
 =======================
-This is the content for multilingual
\ No newline at end of file
+
+Joomla provides robust support for multiple languages, allowing developers to create extensions that can be easily translated and used in different languages.
+
+## How Joomla Supports Multiple Languages
+
+Joomla differentiates between **installed languages** (languages that the Joomla core or extensions are translated into) and **content languages** (languages that site content can be written in). 
+
+This distinction helps in managing translations more effectively.
+
+### Language Constants
+
+To handle translations, Joomla uses **language constants**. These constants are defined in language files (`.ini` and `.sys.ini`) and are used throughout the codebase. 
+
+By using **constants**, you can ensure that your extensions are easily translatable.
+
+### Language Files
+
+**Language files** are simple text files that map language constants to their translations. There are two main types of language files:
+- `.ini` files: Used for **regular translations.**
+- `.sys.ini` files: Used for **installation and system messages.**
+
+Here’s an example of a language file (`en-GB.mod_example.ini`):
```

Comment:
In Joomla 4 and 5 there is no need to add prefix en-GB to the language files anymore.